packages/meditrak-server/src/apiV2/CRUDHandler/helpers.js

```diff
@@ -3,8 +3,8 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-// if the endpoint is /surveys/5a5d1c66ae07fb3fb025c3a3/answers, the resource is 'surveys'
-export const extractResourceFromEndpoint = endpoint => endpoint.split('/')[1];
+// if the endpoint is surveys/5a5d1c66ae07fb3fb025c3a3/answers, the resource is 'surveys'
+export const extractResourceFromEndpoint = endpoint => endpoint.split('/')[0];
```

Comment:
After this change, `req.endpoint` changes from e.g. `/surveys` to `surveys`, which was an unintentional byproduct of using `.split` in the middleware, but one I quite like.